src/blocks/accordion/styles/style.scss

```diff
@@ -14,3 +14,11 @@ _:-webkit-full-screen,
 .wp-block-coblocks-accordion summary {
 	display: block;
 }
+
+.edit-post-visual-editor {
+	.wp-block-coblocks-accordion-item {
+		.wp-block-coblocks-accordion-item__title {
```

Comment:
This CSS affects the editor and should be moved to the editor.scss file. This should likely be in the same file as the dashicons CSS.